Formula/mapserver.rb

```diff
@@ -47,7 +47,6 @@ def install
       -DWITH_WFS=ON
       -DWITH_FRIBIDI=OFF
       -DWITH_HARFBUZZ=OFF
-      -DPYTHON_EXECUTABLE:FILEPATH=#{which("python")}
```

Comment:
I'm not sure removing this is the right thing to do. These were generally added for good reason typically having to do with something deciding to use python3 just because it was installed. If anything it could be replaced with /usr/bin/python, but I don't think that has anything to do with fixing the test.

Reply:
I think I removed it because it didn't seem to be doing anything. IIRC, and I may well not do, Ruby doesn't do anything with that `#{which("python")}` because it is inside a `%w[` rather than a `%W[`. I thought about fixing it but figured that would possibly demand new bottles, which I was trying to avoid here to an extent.

Reply:
(Although I agree it doesn't match the commit message)

Reply:
the %w is clearly a bug

Reply:
Indeed. I wondered if since that line wasn't getting executed as-expected it might just be superfluous, and thus tried that way instead. Was a 33/33/33 toss-up on what to do with it.

Reply:
if you can test the build with python3 installed that would be great

Reply:
Yeah, it does indeed blow up if you don't fix it.

```
-- Found PythonInterp: #{which("python")} (found version "1.4")
-- Found PythonLibs: /usr/local/Frameworks/Python.framework/Versions/3.6/lib/libpython3.6m.dylib (found version "3.6.4")

...

-I/usr/local/Frameworks/Python.framework/Versions/3.6/include/python3.6m -I/tmp/mapserver-20180308-39273-1z0lx1h/mapserver-7.0.7/mapscript/swiginc -I/tmp/mapserver-20180308-39273-1z0lx1h/mapserver-7.0.7/mapscript -I/tmp/mapserver-20180308-39273-1z0lx1h/mapserver-7.0.7/mapscript/python  -DNDEBUG -fPIC   -o CMakeFiles/_pythonmapscript.dir/mapscriptPYTHON_wrap.c.o   -c /tmp/mapserver-20180308-39273-1z0lx1h/mapserver-7.0.7/build/mapscript/python/mapscriptPYTHON_wrap.c
/tmp/mapserver-20180308-39273-1z0lx1h/mapserver-7.0.7/build/mapscript/python/mapscriptPYTHON_wrap.c:149:11: fatal error: 'Python.h' file not found
# include <Python.h>
          ^~~~~~~~~~
1 error generated.
make[2]: *** [mapscript/python/CMakeFiles/_pythonmapscript.dir/mapscriptPYTHON_wrap.c.o] Error 1
make[1]: *** [mapscript/python/CMakeFiles/_pythonmapscript.dir/all] Error 2
make: *** [all] Error 2
```

Reply:
So should probably be turned into a `%W[` I guess 🙃.

Reply:
are we having fun yet?

Reply:
Always!

Reply:
and is it fixed if you make it %W

Reply:
Yeah, that just passes `/usr/bin/python` along, which is why I made it explicit in the end.

Reply:
Not sure if it works with Python 2.6 but I don't have a <=10.6 to test with.

Reply:
Yeah we probably need to be using `#!/usr/bin/env python2.7` if we care about that working still. I'm sure it would make Linuxbrew 🐧s happy too.